<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>601</width>
    <height>253</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Cient Application</string>
  </property>
  <widget class="QPushButton" name="loginButton">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>30</y>
     <width>95</width>
     <height>26</height>
    </rect>
   </property>
   <property name="text">
    <string>Login</string>
   </property>
  </widget>
  <widget class="QPushButton" name="logout_Button">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>10</y>
     <width>95</width>
     <height>26</height>
    </rect>
   </property>
   <property name="text">
    <string>Logout</string>
   </property>
  </widget>
  <widget class="QPushButton" name="getfile_Button">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>60</y>
     <width>95</width>
     <height>26</height>
    </rect>
   </property>
   <property name="text">
    <string>Get File</string>
   </property>
  </widget>
  <widget class="QPushButton" name="quit_Button">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>90</y>
     <width>95</width>
     <height>26</height>
    </rect>
   </property>
   <property name="text">
    <string>Quit</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="status_Text">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>160</y>
     <width>591</width>
     <height>91</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
